### PR TITLE
Add missing external functions tests for burn wrapped ajna

### PR DESCRIPTION
**Pull request by prat-gpt**
# Description of change
## High level
* See - https://github.com/Fixed-Point-Solutions/ajna-PositionManager-RewardsManager/issues/14
* Add tests for all external methods

### CODE REVIEW

**Review by MikeHathaway — APPROVED** [review]
LGTM - thanks for the tests

File: test/unit/BurnWrappedToken.t.sol [hunk 1]
```diff
@@ -127,4 +132,157 @@ contract BurnWrappedTokenTest is Test {
...
+        SigUtils.Permit memory permit = SigUtils.Permit({
+            owner: owner,
+            spender: spender,
+            value: tokensToTransfer,
+            nonce: 0,
+            deadline: block.timestamp + 1 days
+        });
+
+        bytes32 digest = _sigUtils.getTypedDataHash(permit);
+        (uint8 v, bytes32 r, bytes32 s) = vm.sign(ownerPrivateKey, digest);
```
Comment (EdNoepel) [inline test/unit/BurnWrappedToken.t.sol:234]: Why can't we just initialize `oldOwner` using `makeAddr` on line 211, and `startPrank` as the old owner here?

**Review by EdNoepel — APPROVED** [review]
One comment, but looks good.

**Review by ith-harvey — APPROVED** [review]
LGTM

**Pull request merged**